packages/web-components/src/components/menu/menu-item-radio-group.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { prefix } from '../../globals/settings';
+import styles from './menu-item.scss?lit';
+import { carbonElement as customElement } from '../../globals/decorators/carbon-element';
+import { ChangeEventHandler } from 'react';
```

Comment:
shouldn't have to import from react if we follow the comment I made below in `menu-item-selectable` about events